internal/blockchain/ethereum/ethereum.go

```diff
@@ -134,6 +134,10 @@ type EthconnectMessageHeaders struct {
 	ID   string `json:"id,omitempty"`
 }
 
+type FFIGenerationInput struct {
```

Comment:
Is this change described in the PR description?

It has migration implications.

Reply:
Thanks I will note that. This is a breaking change which should be called out in the release notes as well.